src/GraphQL/Types/Collections/SchemaDirectives.cs

```diff
@@ -25,6 +25,9 @@ public class SchemaDirectives : IEnumerable<Directive>
         /// </summary>
         public virtual DeprecatedDirective Deprecated { get; } = new DeprecatedDirective();
 
+        // TODO: remove this one when the next version of the spec will be released
+        internal DeprecatedDirective DeprecatedDraft { get; } = new DeprecatedDirective(deprecationOfInputValues: true);
```

Comment:
I do not like it and I will glad to remove it wit new spec release.

Reply:
Waiting for the GraphQL WG to merge anything is really quite hopeless.  It's very sad they do not explicitly state in the spec that graphql clients should not break when built-in types have properties they do not understand, or additional built-in types are present.  This prevents GraphQL itself from adding features to the spec as non-breaking changes.  Instead all the common tooling breaks for each feature like this because of this poor design decision.  It also causes problems when attempting to implement experimental features.  So they won't add features to the spec because nobody supports them, which is because the common tools are explicitly programmed to disallow experimental features.  It's a recursively bad loop.

Reply:
Yep.

Reply:
I agree I don't like this design.  Can we make an internal method to overwrite the value of the public property during schema initialization or something?  Rather than exposing the 'wrong' directive when the experimental feature is enabled?  Honestly I'm not exactly sure how this class is used.

Reply:
Maybe add a public constructor with the property, then in the Schema constructor, it passes the value from Features.whatever.  Then the RegisterDirectives call doesn't need the ternary

Reply:
> Can we make an internal method to overwrite the value of the public property during schema initialization or something?

It would conflict in case of multiple schemas with different settings in single app.

Reply:
I don't think so.  These are not static properties and they are not in a static instance.

See here:

![image](https://user-images.githubusercontent.com/6377684/229572490-952ad6b2-b083-4ca6-8976-4a85042d090d.png)

Just change it to

```cs
Directives = new SchemaDirectives(Features.DeprecationOfInputValues);
Directives.Register(Directives.Include, Directives.Skip, Directives.Deprecated);
```

![image](https://user-images.githubusercontent.com/6377684/229573128-87959725-18a2-4b05-8a8b-7c527c5d14d3.png)

However, I don't understand how the code works either with or without that change.  How can `Features.DeprecationOfInputValues` be changed prior to the code in `Schema`'s constructor be called?  `Features` is not static.

![image](https://user-images.githubusercontent.com/6377684/229573366-f00f1267-e0ef-4b99-bed2-a002c0795fbe.png)

Reply:
You can see from the introspection result test that the wrong directive is being applied.

Reply:
Oops, my bad, fixed.